www/templates/default/html/manager/EditEvent.tpl.php

```diff
@@ -221,6 +222,14 @@ class="dcf-btn dcf-btn-primary"
                     <th class="dcf-txt-right dcf-pr-0" scope="col">Actions</th>
                 </tr>
             </thead>
+
+            <!-- counting the number of instances present in the event-->
```

Comment:
Can you put this comment into php so the outputted HTML does not have the comment

```
<?php //counting the number of instances present in the event ?>
```